voice/bxml/callbacks/transferAnswer.md

```diff
@@ -1,19 +1,38 @@
 {% method %}
 ##  Transfer Answer Event – <Transfer> verb
 When processing a [`<Transfer>`](../verbs/transfer.md) verb, this event is sent when a called party (B-leg) answers.  The event is sent to
-  the endpoint specified in the `transferAnswerUrl` attribute of the `<PhoneNumber>` tag that answered.  [`<PlayAudio>`](../verbs/playAudio.md) and/or [`<SpeakSentence>`](../verbs/speakSentence.md) verbs returned by this callback will be
+  the endpoint specified in the `transferAnswerUrl` attribute of the `<PhoneNumber>` tag that answered. Accepted verbs returned by this callback will be
   executed for the called party only. Afterward, the called party will be bridged to the original
   call.
-  
-  It is important to note that no other BXML verbs may be specified after a Transfer Answer Event is called.
+
+| Verb            | Accepted |
+|-----------------|----------|
+| [Conference](../verbs/conference.md)      | No       |
+| [Forward](../verbs/forward.md)         | No       |
```

Comment:
It might look better if this table was below the expected response and properties? Not sure

Reply:
I think it's better up top since it's more event-specific than the properties and expected response, since they're on every callback page and have a lot of overlap.

Reply:
Yeah good point